mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Exsom.Mixfile do
 
   defp deps do
     [{:earmark, "~> 0.1", only: :dev},
-     {:erlsom, "~> 1.4"},
+     {:erlsom, git: "https://github.com/willemdj/erlsom.git"},
```

Comment:
We may want to set this to the current commit, so that it doesn't break if they make a major change on `master`